### ISSUE TITLE
Can not use with watch

### ISSUE THREAD

**Issue by goofballLogic** [vgamula/gulp-cucumber#13]
If I use gulp-cucumber in connection with some sort of watch (e.g. gulp.watch or gulp-watch), cucumber doesn't reload the step files when i kick off the new cucumber execution. Internally, cucumber.js is using require() to load the step files, so they are already cached when gulp-cucumber executes the CLI a second time.

**Comment by ludwigm** [comment]
I see this Stackoverflow question regarding this. It is not a nice solution but the thing which was proposed in there worked: https://stackoverflow.com/questions/29931664/reloading-cucumber-js-steps-with-gulp-watch-and-gulp-develop-server#

**Comment by nicosommi** [comment]
+1

**Comment by dcrockwell** [comment]
+1

**Comment by vgamula** [comment]
Hi guys! @goofballLogic @ludwigm @nicosommi @dcrockwell 

I fixed this and published new version 0.0.11 to npm.

You can check how its working now and reopen this issue if you find any problem.

Thanks!

**Comment by jfstephe** [comment]
@vgamula This didn't work for me as when I specified the js files I get:

TypeError: Class constructor <MY_CLASS_NAME> cannot be invoked without 'new'
    at /var/devProjects/bongo/server/node_modules/cucumber/lib/cucumber/cli/support_code_loader.js:67:33

I'm aware of some documented workarounds for this in js land but I'm using typescript and don't really see why I should have to alter the ts code for this. As a result I opted for a solution based on the SO post mentioned above.

Just writing this in the hope it helps someone else in the future.
